gpflow/models/sgpr.py

```diff
@@ -116,7 +116,8 @@ def __init__(self,
                  kernel: Kernel,
                  mean_function: Optional[MeanFunction] = None,
                  inducing_variable: Optional[InducingPoints] = None,
-                 num_latent: Optional[int] = None
+                 num_latent: Optional[int] = None,
+                 noise_variance: int = 1.0,
```

Comment:
once again `float`, not `int`
```suggestion
                 noise_variance: float = 1.0,
```